xivo_test_helpers/asset_launching_test_case.py

```diff
@@ -102,6 +100,21 @@ def rm_containers(cls):
 
     @classmethod
     def start_containers(cls, bootstrap_container):
+        if os.getenv('WAZO_TEST_NO_DOCKER_COMPOSE_PULL', '0') == '0':
```

Comment:
```suggestion
        if os.getenv('WAZO_TEST_NO_DOCKER_COMPOSE_PULL', '1') == '1':
```
Variable name is already negative, so the value should be positive to avoid confusion

Reply:
no the code was right and the value is positive if set and by default it's not set.

Reply:
Yes you right, I propose to switch the logic to be more intuitive